TeamCode/src/main/java/org/firstinspires/ftc/teamcode/core/lib/autonomousControl/MotorVelocityData.java

```diff
@@ -0,0 +1,43 @@
+package org.firstinspires.ftc.teamcode.core.lib.autonomousControl;
+
+import org.firstinspires.ftc.teamcode.robot.constants.AutonomousConstants;
+
+public class MotorVelocityData {
+    public double velocityFrontLeft;
+    public double velocityFrontRight;
+    public double velocityBackLeft;
+    public double velocityBackRight;
+    public MotorVelocityData(){
+
+    }
+    public MotorVelocityData(double vFL,double vFR, double vBL,double vBR){
+        velocityFrontLeft = vFL;
+        velocityFrontRight = vFR;
+        velocityBackLeft = vBL;
+        velocityBackRight = vBR;
+    }
+    public MotorVelocityData updateAppliedVelocities (Pose2d botRelativeVelocitiy){
+        double botVX = botRelativeVelocitiy.getX();
```

Comment:
in order to avoid a verbose code you can use var

Reply:
that can lead to runtime crashes if botRelativeVelocity.getX() is null at any point, which could happen if the webcam malfunctions during a match. It is better to leave it as a double to avoid this edge case